SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -128,6 +130,12 @@ void FSpatialGDKEditorToolbarModule::StartupModule()
 	LocalReceptionistProxyServerManager->Init(GetDefault<USpatialGDKEditorSettings>()->LocalReceptionistPort);
 
 	SpatialGDKEditorInstance = FModuleManager::GetModuleChecked<FSpatialGDKEditorModule>("SpatialGDKEditor").GetSpatialGDKEditorInstance();
+
+	bSpatialDebuggerEditorEnabled = SpatialGDKEditorSettings->bSpatialDebuggerEditorEnabled;
```

Comment:
Is there an advantage on storing a duplicate of this property here instead of just always accessing the value in editor settings?